samples/dashboards/demo/accounting/demo.js

```diff
@@ -0,0 +1,658 @@
+                selected: 2
+            },
+            tooltip: {
+                formatter: function () {
+                    const { x, points } = this;
+                    const format = v => '$' + (v / 1e6).toFixed(2) + 'M';
+                    const color = (s, color) => `
+                        <span class="highcharts-color-${color}">${s}</span>
+                    `;
+                    const date = Highcharts.dateFormat('%B %Y', x);
+
+                    if (x <= currentMonth) {
+                        return `<span style="font-size: 10px">${date}</span><br>
+                            ${color('&#9679;', 0)}&nbsp;
+                            Result: ${color(format(points[0].y), 0)}
+                        `;
+                    }
+
+                    return `<span style="font-size: 10px">
+                            Forecast for ${date}
+                        </span><br>
+                        ${color('&#10138;', 2)}&nbsp;
+                        Optimistically: ${color(format(points[2].y), 2)}<br>
+                        ${color('&#9679;', 0)}&nbsp;
+                        Average: ${color(format(points[0].y), 0)}<br>
+                        ${color('&#10136;', 1)}&nbsp;
+                        Pessimistically: ${color(format(points[1].y), 1)}
+                    `;
+                }
+            },
+            credits: {
+                enabled: false
+            },
+            series: [{
+                name: 'Result',
+                zIndex: 2
+            }, {
+                name: 'Pessimistically'
+            }, {
+                name: 'Optimistically'
+            }]
+        }
+    }]
+}, true);
+
+board.then(res => {
+    const table = res.dataPool.connectors.data.table.modified.columns;
+
+    const revKPI = res.mountedComponents[0].component;
+    const revForecast = res.mountedComponents[1].component;
+    const costKPI = res.mountedComponents[2].component;
+    const costForecast = res.mountedComponents[3].component;
+    const resKPI = res.mountedComponents[4].component;
+    const resForecast = res.mountedComponents[5].component;
+
+    const firstRowID = table.Date.findIndex(d => d === Date.UTC(currentYear));
+    const lastRowID = table.Date.findIndex(d => d === currentMonth);
+    const forecastRowID =
+        table.Date.findIndex(d => d === Date.UTC(currentYear, 11));
+
+    let revYTD = 0,
+        costYTD = 0;
+    for (let i = firstRowID; i <= lastRowID; i++) {
+        revYTD += table.Revenue[i] / 1e6;
+        costYTD += table.Cost[i] / 1e6;
+    }
+
+    let revYearlyForecast = revYTD,
+        costYearlyForecast = costYTD;
+    for (let i = lastRowID + 1; i <= forecastRowID; i++) {
+        revYearlyForecast += table.RevPredA[i] / 1e6;
+        costYearlyForecast += table.CostPredA[i] / 1e6;
+    }
+
+    revKPI.chart.addSeries({ data: [revYTD] });
+    revKPI.chart.subtitle.update({
+        text: `${Math.round(revYTD / revTarget * 100)}% of annual target`
+    });
```

Comment:
Instead of 2 redraws you can call one.

```
revKPI.chart.addSeries({ data: [revYTD] }, false);
revKPI.chart.subtitle.update({...});
```

Reply:
I don't think I can do it in this case because `subtitle.update` doesn't redraw the chart, it only updates the subtitle `SVGElement`.

See: https://jsfiddle.net/BlackLabel/h53uryze/

However, when I look at the implementation of this method, now I am not sure whether this is the correct behavior, cause the redraw argument is passed only to the [`layOutTitles`](https://github.com/highcharts/highcharts/blob/5132ff9c646ad99c494a5d6d82f4d843e1ba17d2/ts/Core/Chart/Chart.ts#L1263) method, in which it is unused.

Anyway, because I'm not sure is this a bug in Highcharts and whether it will be fixed once, I'll change the order to:
1. subtitle with the second argument false
2. second - `addSeries`.